src/components/plugins/plugins-manager/plugins-manager.ts

```diff
@@ -0,0 +1,49 @@
+import {AudioPlugin} from '../../../types/audio-plugin';
+import {FakeEventTarget} from '@playkit-js/playkit-js';
+import {FakeEvent} from '@playkit-js/playkit-js';
+
+export const AUDIO_PLAYER_ACTIVE_PLUGINS_UPDATED = 'audio_player_active_plugins_updated';
```

Comment:
please put event names / enums in a separate file

Reply:
you don’t create an enum for only one field, and since it is an internal event that is not intended for external usage I don’t find it necessary to extract it, it is just overkill and verbose

Reply:
As Sivan suggested, we can assume that futurewise we will have more events, please move this event definition to a separate file